src/gui/file_manager/file_manager.cpp

```diff
@@ -50,8 +60,38 @@ QString file_manager::file_name() const
     return QString();
 }
 
-void file_manager::open_file(const QString& file_name)
+void file_manager::file_successfully_loaded(QString file_name)
 {
+    // autosave every 60s
+    m_timer->start(60 * 1000);
+
+    m_file_name        = file_name;
+    m_shadow_file_name = get_shadow_file(file_name);
+    m_file_watcher->addPath(m_file_name);
+    m_file_open = true;
+    update_recent_files(m_file_name);
+    g_python_context = std::make_unique<python_context>();    // TODO HANDLE PYTHON CONTEXT SEPARATELY
```

Comment:
What about this ToDo?

Reply:
Not mine, was already there, the line was just in a different spot.